README.md

```diff
@@ -2,45 +2,44 @@
 
-This bundle integrates Recommendation services into eZ Platform. It supports the [YOOCHOOSE](https://yoochoose.com/) recommender service.
+This bundle extends the functionality of eZ with a recommendation engine, powered by [YOOCHOOSE](https://yoochoose.com/).  It allows you to track the way visitors use your website and suggests recommended content to them based on their behavior.
 
-**Support** Stable versions of this bundle is officially supported by eZ as of version 1.0.0, for users with a eZ Enterprise and YOOCHOOSE subscription.
+See [YOOCHOOSE documentation](https://yoochoose.com/Personalization-Solution/Documentation) to learn how the recommendation engine works from the YOOCHOOSE side.
 
 ## Requirements
 
 - PHP 5.4.4 *or higher PHP 5.x version*
 - Symfony 2.7 *or higher Symfony 2.x version*
-- eZ Publish v5.4.1+ or eZ Platform/Studio v1.0+, with the REST API configured to use sessions and publicly open to the YOOCHOOSE servers.
+- eZ Publish v5.4.1+ or eZ Platform/Enterprise v1.0+, with the REST API configured to use sessions and publicly open to the YOOCHOOSE servers.
 - A YOOCHOOSE subscription
 
-This bundle is independent from legacy's `ezrecommendation` extension, and doesn't require it.
+This bundle is independent from the `ezrecommendation` extension used in legacy, and does not require it.
 
-## Installation
+## Step 1: Installation
 
-This package is available via composer, so the instructions below are similar to how you install any other open source Symfony Bundle.
+1\. Run the following from your eZ Publish / eZ Platform installation root *(here with most recent 1.0.x release)*:
 
-Run the following from your eZ Publish / eZ Platform installation root *(here with most recent 1.0.x release)*:
 ```bash
 composer require --no-update ezsystems/recommendation-bundle:^1.0.0
 composer update --prefer-dist
 ```
 
-Enable the bundle in `ezpublish/EzPublishKernel.php`:
+2\. Enable the bundle in `ezpublish/EzPublishKernel.php` (for eZ Publish Platform) or `app/AppKernel.php` (for eZ Platform):
 ```php
 $bundles = array(
     // existing bundles
     new EzSystems\RecommendationBundle\EzSystemsRecommendationBundle()
 );
 
 ```
-Import additional routing by adding the following lines to your `routing.yml` file:
+3\. Import additional routing by adding the following lines to your `routing.yml` file:
 
 ```yaml
 recommendationBundleRestRoutes:
     resource: "@EzSystemsRecommendationBundle/Resources/config/routing_rest.yml"
     prefix:   %ezpublish_rest.path_prefix%
 ```
-Keep in mind, that legacy support is disabled by default. To enable legacy search engine (requires `ezpublish-kernel` bundle) uncomment these lines in bundle `services.yml`:
+Keep in mind that legacy support is disabled by default. To enable legacy search engine (requires `ezpublish-kernel` bundle) uncomment these lines in the bundle's `services.yml`:
```

Comment:
Maybe start section with header signifying this step is only relevant for eZ Publish 5.x, or eZ Platform with legacy-bridge + legacy installed.